pages/linux/lxterminal.md

```diff
@@ -0,0 +1,24 @@
+# lxterminal
+
+> Terminal emulator for LXDE.
+> More information: <https://wiki.lxde.org/en/LXTerminal>.
+
+- Open an LXTerminal window:
+
+`lxterminal`
+
+- Open an LXTerminal window, run a command and exit:
```

Comment:
Perhaps this will read better?
```suggestion
- Open an LXTerminal window, run a command, and then exit:
```

Reply:
what about 
```suggestion
- Open an LXTerminal window, run a command then exit:
```

Reply:
I think both `and exit` and `, and then exit` are fine, but `then exit` sounds a little strange.

Reply:
Yes, it's a little hasty.  I like `, and then exit`.